ncollide_testbed2d/src/testbed.rs

```diff
@@ -80,7 +80,7 @@ impl<N: RealField + ToPrimitive> Testbed<N> {
             // font:           fnt,
             // fps:            Fps::new(&fnt),
             grabbed_object: None,
-            grab_anchor: na::origin(),
+            grab_anchor: Point2::origin(),
```

Comment:
This also touches the testbeds, but I was unable to get either of them to compile prior to this pull request. Do you also want to keep the changes for the testbeds in here even though those changes haven't been tested?

Reply:
The testbed change is OK. The testbeds of ncollide are actually not very maintained any more as the nphysics testbed is more advanced. So I guess I'll just remove the ncollide testbed at some point.